src/cljs/org/broadinstitute/firecloud_ui/page/workspace/details.cljs

```diff
@@ -90,7 +90,7 @@
 
 
 (react/defc WorkspaceDetails
-  {:refresh-workspace
+  {:refresh-workspace 
```

Comment:
Please update your editor to remove trailing whitespace. A few of the "edits" in this PR are just the addition of trailing whitespace.